src/provisioner/const.py

```diff
@@ -13,9 +13,22 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+from enum import Enum
+
 SERVICE_NAME = "cortx_setup"
 APP_NAME = "provisioner"
 TMP_LOG_PATH = "/tmp/%s" % APP_NAME
 DEFAULT_LOG_PATH = "/var/log/cortx/%s" % APP_NAME
 DEFAULT_LOG_LEVEL = "INFO"
 SUPPORTED_LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+RELEASE_INFO_PATH = "/opt/seagate/cortx/RELEASE.INFO"
+
+class PROVISIONING_STAGES(Enum):
```

Comment:
You can move this to provisioner class itself, unless other classes need it. Only global const shared across modules should appear here.

Reply:
Done